core/src/script/CGXP/plugins/FeaturesGrid.js

```diff
@@ -276,8 +276,12 @@ cgxp.plugins.FeaturesGrid = Ext.extend(cgxp.plugins.FeaturesResult, {
      */
     csvEncoding: 'UTF-8',
 
-    /** private: attribute[selectAll]
```

Comment:
This line declare that we use a pivate attribute named `selectAll` then she is usefull :-)